data-protocols/dsp/dsp-api-configuration/src/main/java/org/eclipse/edc/protocol/dsp/api/configuration/DspApiConfigurationExtension.java

```diff
@@ -37,25 +38,23 @@
 @Extension(value = DspApiConfigurationExtension.NAME)
 @Provides({ DspApiConfiguration.class })
 public class DspApiConfigurationExtension implements ServiceExtension {
-    
+
     public static final String NAME = "Dataspace Protocol API Configuration Extension";
-    
     public static final String CONTEXT_ALIAS = "protocol";
-    
     public static final String DEFAULT_DSP_CALLBACK_ADDRESS = "http://localhost:8282/api/v1/dsp";
+    @Setting(value = "The address where the connector is configured to receive callbacks from the DSP API", defaultValue = DEFAULT_DSP_CALLBACK_ADDRESS)
```

Comment:
I would describe this as "The endpoint address used by the runtime to receive DSP callbacks"